### ISSUE TITLE
unsupportedrole check fails all invalid roles

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1324]
The following code shouldn't fail the unsupportedrole check:

```html
<input role="foo">
<input role="input">
```

The problem this causes is that these code snippets are reported as:

> Fix all of the following
> - Abstract role cannot be directly used
> - The role used is not widely supported in assistive technologies
